usaspending_api/disaster/v2/views/object_class/spending.py

```diff
@@ -132,11 +132,11 @@ def award_queryset(self):
         )
 
 
-def universal_annotations():
+def shared_annotations():
```

Comment:
"universal" wasn't the best name, but "shared" is very ambiguous. Maybe something more specific like, "object_class_annotations" or something?